commands/addons/create.js

```diff
@@ -2,12 +2,13 @@
 
 const cli = require('heroku-cli-util')
 const co = require('co')
+let waitForAddonProvisioning = require('../../lib/addons_wait')
 
 function parseConfig (args) {
   let config = {}
   while (args.length > 0) {
     let key = args.shift()
-    if (!key.startsWith('--')) throw new Error(`Unexpected argument ${key}`)
+    if (!key.startsWith('--')) { throw new Error(`Unexpected argument ${key}`) }
```

Comment:
Do you know if there's something about the CLI that requires us to do this sort of validation? I thought the CLI just parsed all flags into an object? cc @dickeyxxx